### ISSUE TITLE
Calibre interference

### ISSUE THREAD

**Issue by dabnciencias** [reanimate/reanimate#177]
Hello.
I recently installed Calibre in my system and noticed that whenever I try to view reanimate example animations with stack I get a new window with a Calibre EPUB viewer, and a pop-up window asking me to import an HTML or DOCX file as an EPUB, instead of the usual browser tab -and the preview is nowhere to be seen.

Here is an image of what happens when I try to view an example animation with Calibre installed:

![image](https://user-images.githubusercontent.com/38543893/95256502-862f8380-07e8-11eb-837e-586d45d55231.png)

Uninstalling Calibre fixes the issue, but I'd like to be able to keep that program and view animation previews in-browser. I'm somewhat new to Haskell, but if anyone considers this a "good first issue" and can give me any pointers, I'd greatly appreciate it, since I'd like to try helping out.
Cheers.

**Comment by Lemmih** [comment]
It looks like Callibre has been set up as your default app for HTML documents. If you're using Ubuntu, try going into Settings -> Applications and then tell Callibre to not handle HTML files.

**Comment by dabnciencias** [comment]
That's exactly it. I fixed it by setting Firefox as my default app for HTML documents.

For further reference, I did this in Arch by installing perl-file-mimeinfo, downloading an html file (named, for example, myhtmlfile.html), running the command `mimeopen -d myhtmlfile.html` in the download directory, and then choosing Firefox on prompt.

Thank you for the quick response! This issue has been resolved.
